src/pages/index.js

```diff
@@ -13,17 +13,16 @@ export default function Hello() {
       <img alt="" src="/img/homepage/hero-bg.svg" class={classes.heroBg} />
 
       <div class={`${classes.main} ${classes.container}`}>
-        <h1>The same Actual Budget, but Open Source</h1>
+        <h1>Enjoy managing your finances</h1>
         <p class={classes.heroText}>
-          This website is for the Open Source version of Actual Budget, it is a
-          super fast privacy-focused app for managing your finances. You own
-          your data, and we will sync it across all devices with optional
-          end-to-end encryption.
+          Actual Budget is a super fast privacy-focused app for managing your
+          finances. You own your data and can do with it whatever you want.
```

Comment:
This reads slightly better IMO

```suggestion
          finances. You own your data and can do whatever you want with it.
```

Reply:
Agreed! Great to have native-speakers on the team.